ios/browser/api/url/url_utils.h

```diff
@@ -52,6 +52,29 @@ OBJC_EXPORT
 - (bool)brave_hasScheme:(NSString*)scheme NS_SWIFT_NAME(hasScheme(scheme:));
 @end
 
+OBJC_EXPORT
+@interface NSURL (StaticUtilities)
+
+/// Returns the domain registry
+///  google.co.uk -> co.uk
+///  google.com -> com
++ (NSString*)brave_registryFromHost:(NSString*)host
+    NS_SWIFT_NAME(registry(from:));
```

Comment:
Minor nit on the Swift names: these don't mention the kind of value passing in (host) anywhere. I'd recommend either doing one of these three for all of them:

- `registryFrom(host:)`
- `registry(fromHost:)`
- `registry(host:)`

The last one is probably the simplest.